packages/react/src/components/Address/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface AddressProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+export const Address = ({ children, className, ...other }: AddressProps) => (
+  <address className={classNames('Address', className)} {...other}>
+    {children}
+  </address>
+);
+
+Address.displayName = 'Address';
+Address.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string
+};
+
+interface AddressLineProps {
+  children?: React.ReactNode;
+  className?: string;
+}
+
+export const AddressLine = ({
+  children,
+  className,
+  ...other
+}: AddressLineProps) =>
+  !!children && (
+    <div className={classNames('Address__line', className)} {...other}>
+      {children}
+    </div>
+  );
+
+AddressLine.displayName = 'AddressLine';
+AddressLine.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+interface AddressCityStateZipProps {
+  city: React.ReactNode;
+  state: React.ReactNode;
+  zip: React.ReactNode;
+  className?: string;
+}
```

Comment:
```suggestion
interface AddressCityStateZipProps extends React.HTMLAttributes<HTMLElement> {
  city: React.ReactNode;
  state: React.ReactNode;
  zip: React.ReactNode;
}
```

Reply:
Also, should `country` be supported here?

Reply:
Good question, but no. Country should have its own line: https://faq.usps.com/s/article/International-addressing-requirements